extensions/ql-vscode/src/vscode-tests/cli-integration/data-remote-qlpack/qlpack.yml

```diff
@@ -1,4 +1,4 @@
 name: github/remote-query-pack
 version: 0.0.0
 dependencies:
-  codeql/javascript-all: '*'
+  codeql/javascript-all: '${workspace}'
```

Comment:
I don't know these tests well, but do you think you could add a comment here to explain that the `'${workspace}'` bit is important to ensure tests pass? Not sure that it's clear from the name of the test/directory.